pipeline.py

```diff
@@ -1007,8 +1007,24 @@ def build_image_generic(
             # But since we don't run daily rebuilds on ecr image builds, we can do that step instead here.
             # We only need to push manifests for multi-arch images.
             create_and_push_manifest(registry_address, version, architectures=architectures)
+            latest_tag = "latest"
+            if not is_running_in_patch() and is_running_in_evg_pipeline():
+                logger.info(f"Tagging and pushing {registry_address}:{version} as {latest_tag}")
```

Comment:
q: do you think this as an attribute and trace is useful? Consider the cross-patches query-ability.

Reply:
Added it here: https://github.com/mongodb/mongodb-kubernetes/pull/205/files#diff-6b92e3e77956f8020482ca91889a24f3411c9a4854f2aaaac30ee504ef26ede4R1052